### DISCUSSION TITLE
aws bedrock模型列表显示不全

### DISCUSSION THREAD

**Discussion by imwito** [lobehub/lobehub#2396] [General Question]
请问aws bedrock可以添加个类似openai下获取模型列表的选项吗，目前列表里只有有限的几个。
![aws bedrock](https://github.com/lobehub/lobe-chat/assets/51877240/f3a5635a-9d41-4f02-beb5-be6a1c07b32e)
![openai](https://github.com/lobehub/lobe-chat/assets/51877240/f6a7d5f1-62b6-4fe0-976a-45563c35e423)
十分感谢！

**Comment by arvinxx** [comment]
没列出来的是目前不支持，就算开出来你也用不了
